docs/comb-ssi/logic-gates.md

```diff
@@ -222,3 +222,23 @@ endmodule
 ```
 
 {% include gates.html %}
+
+{:.quiz}
+1. An OR gate has n inputs. One input is high and n-1 are low, output is `____________`
+   1. High
+   * Low
+   * alternately high and low
+2. When an input signal 1 is applied to a `NOT` of `NOT` gate, the output is `____________`
```

Comment:
Since the blank is at the end, `____` is not required.
```suggestion
2. When an input signal 1 is applied to a `NOT` of `NOT` gate, the output is
```